c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -251,29 +251,29 @@ help:
 	@echo
 	@echo "Main targets:"
 	@echo
-	@echo "- build					Build and configure the project"
-	@echo "- checks					Perform a number of checks on the code"
-	@echo "- serve					Run the development server (pserve)"
-	@echo "- clean					Remove generated files"
-	@echo "- cleanall				Remove all the build artefacts"
+	@echo "- build			Build and configure the project"
+	@echo "- checks			Perform a number of checks on the code"
+	@echo "- serve			Run the development server (pserve)"
+	@echo "- clean			Remove generated files"
+	@echo "- cleanall		Remove all the build artefacts"
 	@echo
 	@echo "Utility targets:"
 	@echo
-	@echo "- update					Update the project (GIT, node, pip)"
+	@echo "- update			Update the project (GIT, node, pip)"
 	@echo "- update-git-submodules	Update the GIT submodules"
 	@echo "- update-node-modules	Update node modules (using --force)"
 	@echo
 	@echo "Secondary targets:"
 	@echo
-	@echo "- build-cgxp				Build the javascript and the css for cgxp"
-	@echo "- build-ngeo				Build the javascript and the css for ngeo"
-	@echo "- build-server			Build the files required by the server"
-	@echo "- compile-js-catalog		Compile the angular translation catalog"
-	@echo "- compile-py-catalog		Compile the angular translation catalog"
-	@echo "- flake8					Run flake8 checker on the Python code"
-	@echo "- lint-ngeo				Check the JavaScript code with linters for ngeo"
-	@echo "- template-clean		 	Clean the template file"
-	@echo "- template-generate		Generate the template file"
+	@echo "- build-cgxp		Build the javascript and the css for cgxp"
+	@echo "- build-ngeo		Build the javascript and the css for ngeo"
+	@echo "- build-server		Build the files required by the server"
+	@echo "- compile-js-catalog	Compile the angular translation catalog"
+	@echo "- compile-py-catalog	Compile the angular translation catalog"
```

Comment:
Shouldn't that be `Compile the Python translation catalog` instead of `angular`?